src/lib/types.ts

```diff
@@ -9,7 +9,15 @@ export interface TransactionStats {
   stepsNumberLast7Days: number[]
 }
 
+export interface ContractStats {
```

Comment:
Use camelCase for ContractStats fields names

Reply:
done